openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        period:
+          description: Revenue month.
+          type: string
+          pattern: '^\d{4}-\d{2}$'
+        total:
+          description: The total MRR amount.
+          type: number
+          format: double
+        currency:
+          $ref: ./CurrencyCode.yaml
+        breakdown:
+          description: |
+            MRR could be broken down in several categories, described below.
+            The difference between current period MRR and previous period MRR would be equal to `new + reactivated + expanded - contracted - churned`.
+          type: object
+          properties:
+            new:
+              description: |
+                MRR of new customers who didn't have revenue before they placed an order.
```

```suggestion
                MRR of new customers who didn't have recurring revenue before they placed an order.
```

---

This seems strange. Isn't a "new customer" someone who didn't have revenue?
```suggestion
                MRR of new customers.
```